snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -386,7 +387,10 @@ def ignore(directory, files):
         shutil.copytree(self.code.sourcedir, self.code.build_basedir,
                         symlinks=True, ignore=ignore)
 
+        script_runner = ScriptRunner(builddir=self.code.build_basedir)
+
         self.code.build()
+        script_runner.install(scriptlet=self._part_properties.get('install'))
```

Comment:
You will see in the next PR, it is to avoid a code change once `prepare` is in place.
I will propose it as soon as this lands.